src/lib/services/events-service.ts

```diff
@@ -51,3 +51,23 @@ export const fetchEvents = async (
 
   return events;
 };
+
+export const fetchRawEvents = async (
```

Comment:
We are purposely not running this through data converter, right?

Reply:
yes, for replay functionality we want payloads to stay in the original form

Reply:
Cool. That's what I thought.